<launch>
    <node pkg="pcl_tutorial" type="pcl_tutorial_cloud_node" name="ItsWindyInTheClouds" output="screen">
        <rosparam file="$(find pcl_tutorial)/params/pcl_tutorial_params.yaml" command="load"/>
    </node>
</launch> 